docs/es/README.md

```diff
@@ -1,6 +1,7 @@
 # Algoritmos
 
 ## Ordenamiento
+- [Bubble Sort](./Ordenamiento/Bubble-Sort.md)
 - [Merge Sort](./Ordenamiento/Merge-Sort.md)
 - [Quick Sort](./Ordenamiento/Quick-Sort.md)
```

Comment:
Could you help me to translate the title?

Reply:
Hi @ming-tsai, I've just committed your suggestions. Thank u!

Reply:
```suggestion
- [Ordenamiento de burbuja](./Ordenamiento/Bubble-Sort.md)
- [Ordenar por fusión](./Ordenamiento/Merge-Sort.md)
- [Ordenación rápida](./Ordenamiento/Quick-Sort.md)
```